src/v2/Apps/Order/Routes/Offer/index.tsx

```diff
@@ -168,6 +168,9 @@ export class OfferRoute extends Component<OfferProps, OfferState> {
     }
 
     const listPriceCents = this.props.order.totalListPriceCents
+    const artworkPrice = this?.props?.order?.lineItems?.edges?.[0]?.node
+      ?.artwork?.price
+    const isPriceHidden = isNil(artworkPrice) || artworkPrice === ""
```

Comment:
I'm not sure if you need `isNil` here 🤔 
In theory, if the price is `null`, `undefined`, or `""` then just doing `!artworkPrice` should return the expected value, no?